### DISCUSSION TITLE
One small problem

### DISCUSSION THREAD

**Discussion by Eric2793** [3kh0/ext-remover#632] [Fixes JOIN TITANIUM NETWORK TO SHARE PLEASE]
For iBoss, when I click the link as directed, I am met with a frustrating message:
![image](https://user-images.githubusercontent.com/132301310/235704575-2576ce46-9f79-41b1-ac4c-0670802ae257.png)
The Page Is Blocked!

Is there some way to bypass this?
Or is there any other way to disable iBoss?

PS: I tried experimenting to see what would happen if I would go to the link meant for Blocksi, and as I predicted, It was blocked too, but a different screen, showing the Chrome version of a page being blocked
-_-

Edit: Inspect element is disabled on the page for iBoss but not the page meant for Blocksi, so nothing will work on the page for iBoss 
![image](https://user-images.githubusercontent.com/132301310/235706604-9cb7fabb-46e6-43f7-a3e1-fa577dacee46.png)

Edit again: What even is the Blue hat crew? Is it referring to iBoss? When I clicked the Blue Hat crew message which is actually linked to some page, it redirected me to an about:blank page.

**Comment by 3s7vn** [comment]
im pretty sure you run the bookmarklet on the about:blank page

**Reply by Eric2793** [reply]
Tried it, for some reason it closed, ran the bookmarklet on the blocked page also to see what would happen, guess what? 

Nothing.

Absolutely nothing happened. I was expecting something to happen, when I went to the about:blank page and ran the bookmarklet, nothing appeared in console or anywhere

**Reply by amelia-bartlett** [reply]
bookmarklets cannot be executed on extension pages

**Reply by AshtonDavies** [reply]
> bookmarklets cannot be executed on extension pages

Point-blank allows executing them on extension pages I'm pretty sure.

**Comment by 3s7vn** [comment]
but if your school put "javascript://" on the blocklist, your screwed

**Reply by Eric2793** [reply]
I don't think that's the case…

Are there any other options?

Edit: I'm still heavily confused where it says for the reason the page is blocked, “the blue hat crew is the best”.

**Comment by NaniIsNano** [comment]
there are some methods that may work, such as using a proxy or enabling a soft override if it has been enabled by the administrator

The Blue Hat Crew is a group of hackers who specialize in finding vulnerabilities in Microsoft products, though, it's not related to iBoss.

**Reply by Eric2793** [reply]
Oh, thanks, I didn't know this.

**Reply by Eric2793** [reply]
Big problem is I can't use a proxy, as it is disabled, and I think it won't work because of iBoss.

**Reply by NaniIsNano** [reply]
Then it might not be possible to bypass iBoss, though try enabling a soft override if it is enabled by administrator.

**Reply by Eric2793** [reply]
I don't think I can do it on my side, It can only be available for admin. The only way to add a soft override is to sign in as admin, so only the admin can do that.

![image](https://user-images.githubusercontent.com/132301310/235788251-087c1f9d-89bd-4b84-949f-a8552ebacc48.png)

Once again really frustrating!

**Comment by amelia-bartlett** [comment]
Point-blank gives you a context where you can execute bookmarklets, however the extension page is accessed indirectly from that context.  You are not actually on the extension page when executing the bookmarklet.

**Reply by AshtonDavies** [reply]
Yes you are correct. The admin extension launches a privileged page that you can perform management commands on, even though technically not on an extension page it's almost the same thing.

But, there probably isn't an iBoss method for it.
